cypress/integration/dib-travel/company-employees/employees.spec.ts

```diff
@@ -0,0 +1,147 @@
+import { Employee } from '../../../models/employee/employee';
+
+describe('Company Employees - Employees Page', () => {
+  let employeeDetails: Employee;
+
+  before(() => {
+    cy.fixture('company-employees/employees').then((employee) => {
+      employeeDetails = employee;
+    });
+  });
+
+  beforeEach(() => {
+    cy.login();
+    cy.get('dib-layout dib-hamburger-icon').click();
+    cy.get('[routerLink="/people-management/employees"]').click();
+  });
+
+  it('should allow regular user to add new employee', () => {
+    cy.intercept('GET', '/api/secure/v1/corporations/*/employees').as('getCorporationsEmployees');
+    cy.get('dib-people-management new-dib-employees ui-button button').click();
+    cy.get('.cdk-overlay-container dib-employee-dialog ui-input input[name="firstName"]').type(
+      employeeDetails.firstName
+    );
+    cy.get('.cdk-overlay-container dib-employee-dialog ui-input input[name="lastName"]').type(employeeDetails.lastName);
+    cy.get('.cdk-overlay-container dib-employee-dialog ui-input input[type="email"]').type(employeeDetails.email);
+    cy.get('.cdk-overlay-container dib-employee-dialog ui-button button').click();
+    cy.wait('@getCorporationsEmployees').then(() => {
+      cy.reload();
```

Comment:
If this `cy.reload()` really needed, we should double-check the requirements. Maybe this behavior is buggy, and results should be displayed immediately.

Reply:
Did you check this?

Reply:
I checked this and if I delete cy.reload(), the test fails. From my point of view, we need to use reload() because the inserted data is not displayed immediately but after refreshing the page